### PR TITLE
bug(UIS-68): fix JDK version error on hekoru deploy

### PR DESCRIPTION
**Pull request by ElricVermaut**

### CODE REVIEW

File: system.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+java.runtime.version=17
```
Comment (StaceyJin) [inline system.properties:1]: Why was this addition necessary?

**Review by StaceyJin — APPROVED** [review]
Single line fix, appears in order.

File: system.properties (hunk 1 above)
Comment (ElricVermaut) [reply system.properties:1]: apparently hekoru needs the system properties file to tell him which jdk version to use
